scraper.rb

```diff
@@ -1,3 +1,55 @@
-require File.join(File.dirname(__FILE__), "planningalerts_xml_feed", "planningalerts_xml_feed")
+require 'scraperwiki'
+require 'mechanize'
+
+agent = Mechanize.new
+
+def extract_from_p(contents, name)
+	contents.at('p:contains("'+name+'")').inner_text.split(name).last.strip
```

Comment:
`+` usually has a space around it so the line would be:

    contents.at('p:contains("' + name + '")').inner_text.split(name).last.strip

You could also use string interpolation to make it more readable but having to escape the double quotes negates that so the above is probably best.